kuma/wiki/migrations/0008_auto_20190914_2209.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.23 on 2019-09-14 22:09
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+DISCARDED_LOCALES = (
+    'af', 'az', 'cs', 'ee', 'ff', 'fy-NL', 'ga-IE', 'ha', 'hr', 'ig', 'ka',
+    'ln', 'mg', 'ml', 'ro', 'son', 'sq', 'sr', 'sr-Latn', 'sw', 'ta', 'te',
+    'tl', 'tn', 'wo', 'xh', 'yo', 'zu'
+)
+
+
+def delete_documents_within_discarded_locales(apps, schema_editor):
+    """
+    Delete all documents and document-deletion logs within the
+    DISCARDED_LOCALES.
+    """
+    Document = apps.get_model('wiki', 'Document')
+    docs_to_delete = Document.objects.filter(locale__in=DISCARDED_LOCALES)
+    docs_to_delete.update(parent_topic=None)
+    docs_to_delete.delete()
+    DocumentDeletionLog = apps.get_model('wiki', 'DocumentDeletionLog')
+    DocumentDeletionLog.objects.filter(locale__in=DISCARDED_LOCALES).delete()
```

Comment:
Ditto here about the idea about using `.exclude(locale__in=settings.AVAILABLE_LOCALES)`

Reply:
Same here. I'd prefer to keep it as is, since it captures exactly what this data migration should do at this point in time.